shielddash/api/models.py

```diff
@@ -1,8 +1,34 @@
 from django.db import models
 
 
-class Retention(models.Model):
+class Study(models.Model):
+    name = models.CharField(max_length=255, primary_key=True)
+    start_time = models.DateTimeField(blank=True, null=True)
+    end_time = models.DateTimeField(blank=True, null=True)
+
     class Meta:
-        db_table = 'retention'
-    client_id = models.UUIDField()
-    study = models.CharField(max_length=255)
+        db_table = 'studies'
+        verbose_name = 'study'
+        verbose_name_plural = 'studies'
+
+    def __unicode__(self):
+        return self.name
+
+
+class Ping(models.Model):
+    channel = models.CharField(max_length=255)
+    variation = models.CharField(max_length=255)
+    first_run = models.DateTimeField(blank=True, null=True)
```

Comment:
I took first run out of the data. It's only used to calculate the `seen%d` variables as far as I could tell.